packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-index.js

```diff
@@ -361,16 +365,33 @@ test.serial('writeRecords() writes records to Dynamo and PostgreSQL', async (t)
   );
 });
 
-test('Lambda sends message to DLQ when writeRecords() throws an error', async (t) => {
+test.serial('Lambda sends message to DLQ when writeRecords() throws an error', async (t) => {
   const {
     handlerResponse,
-    sqsEvent,
   } = await runHandler({
     ...t.context,
     cumulusMessage: { fail: true },
   });
 
-  t.is(handlerResponse[0][1].body, sqsEvent.Records[0].body);
+  t.is(handlerResponse.batchItemFailures.length, 0);
+  const {
+    numberOfMessagesAvailable,
+    numberOfMessagesNotVisible,
+  } = await getSqsQueueMessageCounts(t.context.queues.deadLetterQueueUrl);
```

Comment:
:100: this is a really good test improvement.

Reply:
Nit: Should we consider evaluating that correct values got put into the queue message?